tests/console/consoletest_setup.pm

```diff
@@ -31,10 +31,13 @@ sub run() {
     # Export the existing status of running tasks for future reference (fail would export it again)
     script_run "ps axf > /tmp/psaxf_consoletest_setup.log";
 
-    # openSUSE 13.2's systemd has broken rules for virtio-net, not applying predictable names (despite being configured)
+    # openSUSE 13.2's (and earlier) systemd has broken rules for virtio-net, not applying predictable names (despite being configured)
     # A maintenance update breaking networking names sounds worse than just accepting that 13.2 -> TW breaks
     # At this point, the system has been upadted, but our network interface changes name (thus we lost network connection)
-    if (check_var('HDDVERSION', "openSUSE-13.2")) {    # copy eth0 network config to ens4
+    if (   check_var('HDDVERSION', "openSUSE-12.1")
+        || check_var('HDDVERSION', "openSUSE-12.2")
+        || check_var('HDDVERSION', "openSUSE-12.3")
+        || check_var('HDDVERSION', "openSUSE-13.2")) {    # copy eth0 network config to ens4
```

Comment:
try this:
if (grep { $_ eq get_var('HDDVERSION') } qw/openSUSE-12.1 openSUSE-12.2 openSUSE-12.3 openSUSE-13.2/)